addons/account/views/res_partner_bank_views.xml

```diff
@@ -86,6 +86,16 @@ action = model.setting_init_bank_account_action()
             <field name="groups_id" eval="[(4, ref('account.group_account_manager'))]"/>
         </record>
 
+        <record id="action_new_credit_card_setting" model="ir.actions.server">
+            <field name="name">Add a Credit Card Account</field>
+            <field name="model_id" ref="model_res_company"/>
+            <field name="state">code</field>
+            <field name="code">
+action = model.setting_init_credit_card_account_action()
```

Comment:
We have to define this function in the account module. Without doing that, users without the `account_online_synchronization` module will have a traceback.